examples/generate_config.sh

```diff
@@ -75,10 +75,10 @@ dtype="\
     },"
 fi
 
-if [ $ZERO_STAGE == 3 ]; then
+if [ $ZERO_STAGE == 2 ]|| [ $ZERO_STAGE == 3 ]; then
 zero="\
     \"zero_optimization\": {
-      \"stage\": 3,
+      \"stage\": $ZERO_STAGE,
       \"reduce_scatter\": false,
       \"stage3_max_live_parameters\": 3e9,
```

Comment:
Does it make sense to sett stage3_xxx parameters for ZeRO2?

Consider to separate the ZeRO 1/2/3